### PR TITLE
test: BackTop js => ts

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog
test: BackTop js => ts

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by MadCcc** [comment]
好高产

**Comment by li-jia-nan** [comment]
> 好高产

周五啦，回家摆烂咯

File: components/back-top/__tests__/index.test.tsx [hunk 1]
```diff
@@ -26,23 +26,27 @@ describe('BackTop', () => {
...
   });
 
   it('invalid target', async () => {
     const onClick = jest.fn();
     const { container } = render(
-      <BackTop onClick={onClick} visible target={() => ({ documentElement: {} })} />,
+      <BackTop
+        onClick={onClick}
+        visible
+        target={() => ({ documentElement: {} } as unknown as HTMLElement)}
```
Comment (MadCcc) [inline components/back-top/__tests__/index.test.tsx:46]: 这个能给个类型正确的吗，顺便可以把 TS 测了

Comment (li-jia-nan) [reply components/back-top/__tests__/index.test.tsx:46]: > 这个能给个类型正确的吗，顺便可以把 TS 测了

done

File: components/back-top/__tests__/index.test.tsx [hunk 2]
```diff
@@ -26,23 +26,21 @@ describe('BackTop', () => {
...
     scrollToSpy.mockRestore();
   });
 
   it('invalid target', async () => {
     const onClick = jest.fn();
-    const { container } = render(
-      <BackTop onClick={onClick} visible target={() => ({ documentElement: {} })} />,
-    );
-    fireEvent.click(container.querySelector('.ant-back-top'));
+    const { container } = render(<BackTop onClick={onClick} visible target={undefined} />);
```
Comment (MadCcc) [inline components/back-top/__tests__/index.test.tsx:42]: 和原来一样填个元素进去吧

Comment (li-jia-nan) [reply components/back-top/__tests__/index.test.tsx:42]: > 和原来一样填个元素进去吧

这个 case 不是测无效的 target 么？填个元素不是就变成有效的了嘛？

**Pull request closed without merging**